app/javascript/shared/transfersLayouts.js

```diff
@@ -11,6 +11,7 @@
 //   targetWell: <position string>
```

Comment:
If it is because of the validations happening in the vue component, and the duplicates finding happening in this part; yes, I think that would be more part of a future refactor for all this transferLayouts in the file. Maybe we could do this during the vue3 upgrade